x-pack/test/plugin_api_integration/plugins/event_log/server/init_routes.ts

```diff
@@ -0,0 +1,286 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import {
+  RequestHandlerContext,
+  KibanaRequest,
+  KibanaResponseFactory,
+  IKibanaResponse,
+  IRouter,
+  Logger,
+  RouteValidationResultFactory,
+} from 'kibana/server';
+import { IEventLogService, IEventLogger } from '../../../../../plugins/event_log/server';
+import { IValidatedEvent } from '../../../../../plugins/event_log/server/types';
+
+export const logEventRoute = (router: IRouter, eventLogger: IEventLogger, logger: Logger) => {
+  router.post(
+    {
+      path: `/api/log_event_fixture/{id}/_log`,
+      validate: {
+        // removed validation as schema is currently broken in tests
+        // blocked by: https://github.com/elastic/kibana/issues/61652
+        params: (value: any, { ok }: RouteValidationResultFactory) => ok(value),
+        body: (value: any, { ok }: RouteValidationResultFactory) => ok(value),
+      },
+    },
+    async function(
+      context: RequestHandlerContext,
+      req: KibanaRequest<any, any, any, any>,
+      res: KibanaResponseFactory
+    ): Promise<IKibanaResponse<any>> {
+      const { id } = req.params as { id: string };
+      const event: IValidatedEvent = req.body;
+      logger.info(`test fixture: log event: ${id} ${JSON.stringify(event)}`);
+      try {
+        await context.core.savedObjects.client.get('event_log_test', id);
+        logger.info(`found existing saved object`);
+      } catch (ex) {
+        logger.info(`log event error: ${ex}`);
+        await context.core.savedObjects.client.create('event_log_test', {}, { id });
+        logger.info(`created saved object`);
+      }
+      eventLogger.logEvent(event);
```

Comment:
we could make use of the logger timing functions here, to make sure the start/stop/duration fields get set - we'd want to insert a delay of maybe 500ms or so between the timer start and stop, and then we can test that the time in the event document is > 500ms and less than some much larger value (10 seconds or so? )